src/language.rs

```diff
@@ -4,7 +4,7 @@ use std::path::{Path, PathBuf};
 use crate::{FormatterError, FormatterResult, IoError};
 
 /// The languages that we support with query files.
-#[derive(Clone, Copy, Debug, PartialEq)]
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
```

Comment:
Is this just for the sake of readers: `Eq` denotes an equivalence relation, but it's not something that the compiler proves to be true?

Reply:
See https://rust-lang.github.io/rust-clippy/master/#derive_partial_eq_without_eq